api/redhatopenshift/HcpCluster.Management/hcpCluster-models.tsp

```diff
@@ -90,6 +90,25 @@ model HcpOpenShiftClusterProperties {
    */
   @visibility("read")
   issuerUrl: url;
+
+  /** Configure cluter capabilities. */
+  @visibility("create", "read")
+  capabilities?: ClusterCapabilitiesProfile;
+}
+
+/** Cluster capabilities configuration. */
+model ClusterCapabilitiesProfile {
+  /** Disabled cluter capabilities. */
+  @visibility("create", "read")
+  disabled?: OptionalClusterCapability[];
+}
+
+/** Cluster capabilities that can be disabled. */
+union OptionalClusterCapability {
```

Comment:
Is there a reason to call this "Optional"? Is that convention or just preference?

Reply:
This is not an exhaustive list of capabilities. Only the capabilities defined as optional by HCP.